ci_framework/roles/tempest/tasks/tempest-tests.yml

```diff
@@ -92,6 +92,6 @@
         dest: "{{ cifmw_tempest_artifacts_basedir }}/exclude.txt"
         content: "{% for test in cifmw_tempest_tests_skipped %}{{ test }}\n{% endfor %}"
 
-    - name: Show tests to be executed
```

Comment:
so you'd rather want to print only the excluded ones? Maybe printing both listing would be better?

Reply:
Please check the previous block, or rather the whole file. This file contains a set of 4 blocks, the first two handle the allowed and skipped tests when a skiplist is specified, the second two handle the allowed and skipped tests when a direct list of tests is specified. The last block printed the wrong information before this patch.

Reply:
heh, probably a wild copy/paste - fine!